### DISCUSSION TITLE
UserWarning when renaming DataArray coordinates

### DISCUSSION THREAD

**Discussion by mgorfer** [pydata/xarray#7950] [Q&A] [answered]
I have code which renames a DataArray and also coordinates in that DataArray
```
        out = da_pressure_levels_monthly.rename(alt_var_name).rename(
            **{
                "pressure": pres_var_name,
                "latitude_bins": lat_var_name,
                "time": time_var_name,
            }
        )
```
Recently, I am getting the UserWarning
`UserWarning: rename 'time' to 'time' does not create an index anymore. Try using swap_dims instead or use set_index after rename to create an indexed coordinate.`

I don't want to create an index, just renaming is okay in my use case. Is there anything I should change in my code? 
Or anything I can do to silence the warning if things are looking good in my code?

**Comment by mgorfer** [comment]
Dear everyone,
I want to bump this and ask for the best approach for my `UserWarning`.

**Reply by benbovy** [reply, marked as answer]
Hi @mgorfer, since all you want is just renaming, you can safely [ignore this warning](https://docs.python.org/3/library/warnings.html#warnings.filterwarnings), which will be dropped eventually.

**Reply by mgorfer** [reply]
Thank you for your reply! 
Then I will just leave it at this and wait for the warning to be dropped.
